securedrop/tests/test_source.py

```diff
@@ -204,6 +204,38 @@ def test_user_must_log_in_for_protected_views(self, source_app):
             text = resp.data.decode('utf-8')
             assert "Enter Codename" in text
 
+    def test_login_with_whitespace(self, source_app):
+        """
+        Test that codenames with leading or trailing whitespace still work"""
+
+        def login_test(app, codename):
+            resp = app.get('/login')
+            assert resp.status_code == 200
+            text = resp.data.decode('utf-8')
+            assert "Enter Codename" in text
+
+            resp = app.post('/login', data=dict(codename=codename),
+                            follow_redirects=True)
+            assert resp.status_code == 200
+            text = resp.data.decode('utf-8')
+            assert "Submit Materials" in text, text
```

Comment:
If I'm not mistaken the idea here is to display `text` in case the assert fails but ... `pytest` instruments the `assert` and displays the actual values being compared when it fails. So +1 on the Nit ;-)

Reply:
Honestly, we should probably do that with all cases since seeing `...` in the error output is not very helpful, but that one probably got left in by mistake.

Reply:
Fixed